### ISSUE TITLE
[Android] ProductVideo: Controls inside video player are only accessible using swipe gesture and not through keyboard.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6416]
### Target Platforms

Android

### SDK Version

Application Version: 2021.09.03.1 (38680868)

### Application Name

Adaptive Cards Android Visualizer

### Problem Description

[ADO 36148776](https://microsoft.visualstudio.com/OS/_workitems/edit/36148776)

**Repro Steps:**

1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'ProductVideo'.
8. Navigate to the video player and activate it.
9. Navigate the controls present inside using swipe gesture.
10. Observe the issue.

**Actual Result:**
Controls inside video player are only accessible using swipe gesture and not through keyboard. The controls are not accessible either by tab or arrow keys.

**Expected Result:**
Controls inside video player should be accessible using keyboard. Either the tab or arrow keys should be able to access the controls

### Card JSON

```JSON
ProductVideo.json
```
